ATOI and ITOA/atoi_itoa.py

```diff
@@ -0,0 +1,43 @@
+'''
+atoi - convert a string to an integer
+itoa - convert an integer to a string
+'''
+
+class atoi_itoa:
+
+    def atoi(self,inp):
+        if len(inp) == 0:
+            return 0
```

Comment:
If I call this function and I get 0 as a result, how do I know whether it was an error or whether the (correct) result of calling atoi("0")